test/components/general/header.test.js

```diff
@@ -16,15 +16,17 @@ describe("Header Component Tests", () => {
     const wrapper = shallow(<Header />);
 
     const expectedProps = [
-      { href: "forum", passHref: true, text: "Forum" },
-      { href: "browse", passHref: true, text: "Browse" },
-      { href: "create", passHref: true, text: "Create A Project" },
-      { href: "signin", passHref: true, text: "SIGN IN", type: "button" }
+      { href: "forum" },
+      { href: "browse" },
+      { href: "create" },
+      { href: "signin", type: "button" }
     ];
+    const expectedTexts = ["Forum", "Browse", "Create A Project", "SIGN IN"];
```

Comment:
Does this test still pass with the capitalized text?

Reply:
It did not, made a change so it does. I noticed I forgot tests for the link component, so i'll add that later and add tests for the correct text-transform class